www/pages/brand-assets.tsx

```diff
@@ -8,15 +8,33 @@ import Link from 'next/link'
 import { useRouter } from 'next/router'
 
 import SectionContainer from '~/components/Layouts/SectionContainer'
+import { NextSeo } from 'next-seo'
 
 const { Title, Text } = Typography
 
 const Index = () => {
   // base path for images
   const { basePath } = useRouter()
 
+  const meta_title = 'Branding | Supabase'
+  const meta_description = 'Get Supabase Brand assets here.'
+
   return (
     <>
+      <NextSeo
+        title={meta_title}
+        description={meta_description}
+        openGraph={{
+          title: meta_title,
+          description: meta_description,
+          url: `https://supabase.com/auth`,
```

Comment:
use `https://supabase.com/${router.pathname}`

This is so it uses the correct URL path.

you'll also need to add `const router = useRouter()` above so you can use `router`

Reply:
All the pages need this updating I think